tests/v2/validation/charts/cisbenchmark_test.go

```diff
@@ -0,0 +1,261 @@
+package charts
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	management "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	v1 "github.com/rancher/rancher/tests/framework/clients/rancher/v1"
+	"github.com/rancher/rancher/tests/framework/extensions/charts"
+	"github.com/rancher/rancher/tests/framework/extensions/clusters"
+	"github.com/rancher/rancher/tests/framework/pkg/config"
+	namegen "github.com/rancher/rancher/tests/framework/pkg/namegenerator"
+	"github.com/rancher/rancher/tests/framework/pkg/session"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+const (
+	// Project that charts are installed in
+	cisbenchmarkProjectName = "cis-operator-system"
+	clusterScanResourceType = "cis.cattle.io.clusterscan"
+	clusterScanReportType   = "cis.cattle.io.clusterscanreport"
```

Comment:
I think these three can be in extensions/charts part that way we can import them if needed

Reply:
agreed

Reply:
+1